### PR TITLE
App router migration root layout

### PR DESCRIPTION
**Pull request by boodland**
Parent PR for the progressive migration of _app.tsx and _document.tsx within the Root Layout

### Issue link / number:
#1083

### What changes did you make?

1. Created basic root layout
2. Added metadata to root layout
3. Created public testing page to check migration

### Why did you make the changes?
In order to be able to migrate the product from pages router to app router and benefits of all the new features available in the app router

### Did you run tests?
Yes

Navigate to http://localhost:3000/public-route-test to check that title, charset and viewport meta tags are added properly

image

### CODE REVIEW

**Comment by boodland** [comment]
@annarhughes I think better to create a app-router-migration branch so I can point all PR to that one, let me know when it is created to edit the PR so it points to app-router-migration

**Comment by annarhughes** [comment]
Hey @boodland are you able to create that parent branch yourself, or even use this branch as the base/parent branch?
I'm not sure if you'd have the correct access if we were to create a branch from the origin repo

**Comment by boodland** [comment]
> Hey @boodland are you able to create that parent branch yourself, or even use this branch as the base/parent branch? I'm not sure if you'd have the correct access if we were to create a branch from the origin repo

Hi @annarhughes, I already tried to use this branch as the parent branch and all the PRs are created in my repo, not visible to Chayn and without Chayn's PR template. I don't think I am able to create a branch in Chayn repo, so please create one so I can point to and carry on with the development. Thanks!

**Comment by boodland** [comment]
@eleanorreem, @annarhughes  I have just changed the base branch, have a look when possible. Thanks

File: app/layout.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+export default function RootLayout({
+  // Layouts must accept a children prop.
+  // This will be populated with nested layouts or pages
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  return (
+    <html lang="en">
```
Comment (eleanorreem) [inline app/layout.tsx:17]: What the impact of setting lang="en" on a multi-lingual site? Might be worth making a TODO to address this later?

File: next-env.d.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 /// <reference types="next" />
 /// <reference types="next/image-types/global" />
+/// <reference types="next/navigation-types/compat/navigation" />
```
Comment (eleanorreem) [inline next-env.d.ts:3]: Hey @boodland what is this line doing? I saw this [issue](/// <reference types="next/navigation-types/compat/navigation" />) but not sure if its related

**Comment by boodland** [comment]
@eleanorreem could you merge develop within nex-js-migration when possible in order to have the latest version? Thanks!

**Comment by eleanorreem** [comment]
done! Thanks @boodland

**Pull request merged**